localstack/config.py

```diff
@@ -904,7 +904,7 @@ def populate_edge_configuration(
 )
 # PUBLIC: documented upon customer request
 LAMBDA_LIMITS_CREATE_FUNCTION_REQUEST_SIZE = int(
-    os.environ.get("LAMBDA_LIMITS_CREATE_FUNCTION_REQUEST_SIZE", 69_905_067)
+    os.environ.get("LAMBDA_LIMITS_CREATE_FUNCTION_REQUEST_SIZE", 70_167_211)
```

Comment:
Why did this change, and where does the new value come from?

Reply:
It comes from the updated snapshots.

AWS changes the behavior related to these limits. Therefore, I added an extra test and adjusted the tests to cover special cases (e.g., zip within limit but request size exceeding).

My guess: The previous error message was too confusing. People uploaded large zips and got a message about request size:
`Request must be smaller than 70167211 bytes for the CreateFunction operation`

Now they check the zip (if relevant) before checking the request size limit leading to a clearer message:
`Zipped size must be smaller than 52428800 bytes`

Reply:
Ah, great! Thanks for clarifying.